### ISSUE TITLE
Date and times issues

### ISSUE THREAD

**Issue by scottjkelly** [negokaz/chrome-google-maps-transit-scheduler#1]
The date and times seem to to be transferring incorrectly from the maps to the calendar. 
They always seem to be in the AM, on the previous day.  
Additionally, it appears to be opening a new tab in Chrome when the calendar app is already on another tab.

**Comment by negokaz** [comment]
@scottjkelly 

> Additionally, it appears to be opening a new tab in Chrome when the calendar app is already on another tab.

This is correct behavior. This extension makes an event with [Google Calendar link](http://useroffline.blogspot.jp/2009/06/making-google-calendar-link.html) without using any APIs.

> The date and times seem to to be transferring incorrectly from the maps to the calendar.
They always seem to be in the AM, on the previous day.

Could you show me a screenshot of maps transit view to reproduce the problem?
